jsonrpc.go

```diff
@@ -1047,6 +1062,8 @@ var rpcHandlers = map[string]RPCHandler{
 	"renewDHCPLease":         {Func: rpcRenewDHCPLease},
 	"keyboardReport":         {Func: rpcKeyboardReport, Params: []string{"modifier", "keys"}},
 	"getKeyboardLedState":    {Func: rpcGetKeyboardLedState},
+	"keypressReport":         {Func: rpcKeypressReport, Params: []string{"key", "press"}},
```

Comment:
The two new functions! `keypressReport` reports when a key is pressed or released, and returns the current KeysDownState (with up to 6 key-rollover, and current modifier state)